Backend/app/kafka-consumers/src/Consumer/Flow.hs

```diff
@@ -26,21 +27,38 @@ import Data.Function
 import qualified Data.HashMap as HM
 import qualified Data.Map.Strict as Map
 import Environment
+import qualified EulerHS.Language as L
 import qualified EulerHS.Runtime as L
 import qualified Kafka.Consumer as Consumer
+import qualified Kernel.Beam.Types as KBT
 import Kernel.Prelude
+import Kernel.Storage.Queries.SystemConfigs as QSC
+import Kernel.Types.Error
 import Kernel.Types.Flow
-import Kernel.Utils.Common (generateGUID, withLogTag)
+import Kernel.Utils.Common hiding (id)
+import "dynamic-offer-driver-app" Storage.Beam.SystemConfigs ()
 import qualified Streamly.Internal.Data.Fold as SF
 import Streamly.Internal.Data.Stream.Serial (SerialT)
 import qualified Streamly.Prelude as S
 
+-- import Debug.Trace as DT
+
 runConsumer :: L.FlowRuntime -> AppEnv -> ConsumerType -> Consumer.KafkaConsumer -> IO ()
 runConsumer flowRt appEnv consumerType kafkaConsumer = do
   case consumerType of
     AVAILABILITY_TIME -> availabilityConsumer flowRt appEnv kafkaConsumer
     BROADCAST_MESSAGE -> broadcastMessageConsumer flowRt appEnv kafkaConsumer
     PERSON_STATS -> updateCustomerStatsConsumer flowRt appEnv kafkaConsumer
+    LOCATION_UPDATE -> locationUpdateConsumer flowRt appEnv kafkaConsumer
+
+runFlowWithTablesR :: L.FlowRuntime -> AppEnv -> FlowR AppEnv a -> IO a
```

Comment:
if the forever thread inside `startConsumerWithEnv` is working then this is not required, can you test without this once ?

Reply:
if that thread thing is not working then fetch new config using time threshold config not on every message.

Reply:
Its working without this also. 
I have tested and updated the PR.